sdk/cognitivelanguage/ai-language-text/review/ai-language-text.api.md

```diff
@@ -65,9 +65,6 @@ export interface ActionPrebuilt extends ActionCommon {
     modelVersion?: string;
 }
 
-// @public
-export type AgeUnit = string;
```

Comment:
Did you merge the resolutions PR? If so, could you rebase on latest main so these changes go away?

Reply:
I did merge the resolution PR and the resolutions have been removed in the file.